.ci/Makefile

```diff
@@ -213,3 +220,10 @@ run-tests-win:
 		-t run-tests \
 		-i $(NODE_IP_ADDRESS),
 	ssh -i $(SSH_KEY) $(NODE_USER)@$(NODE_IP_ADDRESS) 'powershell "C:\Users\$(NODE_USER)\e2e-testing\.ci\scripts\functional-test.ps1"'
+
+# Make goal to run the tests for the MacOS platform. Used by the CI.
+.PHONY: run-fleet-tests-macos
+run-fleet-tests-macos:
```

Comment:
I wonder if this goal will be misunderstood by the users, as it's CI-only, what do you think of prepending the `ci-` prefix to it? `ci-run-fleet-tests-macos`

Reply:
it's not only `ci-` specific, but wanted to clarify that it's consumed by the CI, ideally anyone could run the same goal locally to reproduce the tests

Reply:
if it's possible to locally invoke the ORKA provider, then it's OK to me, although not sure if that's currently possible, only done at the CI

Reply:
it depends on how to setup the environment, if ESS then:

```bash
### spin up cluster
export CLUSTER_NAME=my-test-cluster
.ci/scripts/deployment.sh 'create'
# set env variables stored in Vault
export ELASTICSEARCH_URL=$(vault read ..../${CLUSTER_NAME})
export KIBANA_URL=$(vault read ....)
...
# run the tests for MacOS
PROVIDER='remote' make -C .ci run-fleet-tests-macos

# destroy cluster
.ci/scripts/deployment.sh 'destroy'
```

Reply:
Ok, now I think I get it: this goal is to be used when running the tests on Mac, no matter if it's an Orka machine (CI time) or a local Mac, right?

Reply:
I'd like to have a make goal that retrieves the variables in the same manner the set-env-* does.

Apart from that, in a follow-up, we can entirely replace the contents of the `create-stack` goal with the creation of the cluster, and this would represent a great step forward to test in Cloud!!